frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx

```diff
@@ -0,0 +1,114 @@
+import FormTemplate from '../../common/FormTemplate/FormTemplate';
+import { Divider, styled } from '@mui/material';
+
+import { IntegrationIcon } from '../IntegrationList/IntegrationIcon/IntegrationIcon';
+import cr from 'assets/img/jira/cr.png';
+import connect from 'assets/img/jira/connect.png';
+import manage from 'assets/img/jira/manage.png';
+import React from 'react';
+import { JiraImageContainer } from './JiraImageContainer';
+
+export const StyledContainer = styled('p')(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(2),
+}));
+
+export const StyledGrayContainer = styled('p')(({ theme }) => ({
+    backgroundColor: theme.palette.grey[100],
+    borderRadius: theme.shape.borderRadiusLarge,
+    padding: theme.spacing(3),
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(1),
+}));
+
+export const StyledIconLine = styled('p')(({ theme }) => ({
+    display: 'flex',
+    marginLeft: theme.spacing(1),
+    alignItems: 'center',
+}));
+
+export const StyledLink = styled('a')({
+    textDecoration: 'none',
+});
+
+export const JIRA_INFO = {
+    name: 'jira',
+    displayName: 'Jira',
+    description:
+        'Create, connect, manage, and approve Unleash feature flags directly from Jira',
+    documentationUrl:
+        'https://docs.getunleash.io/reference/integrations/jira-cloud-plugin-installation',
+};
+
+export const JiraIntegration = () => {
+    const { name, displayName, description, documentationUrl } = JIRA_INFO;
+
+    return (
+        <FormTemplate
+            title={`${displayName}`}
+            description={description || ''}
+            documentationLink={documentationUrl}
+            documentationLinkLabel="Jira documentation"
+        >
+            <StyledContainer>
+                <StyledGrayContainer>
+                    <StyledIconLine>
+                        <IntegrationIcon name={name} /> How does it work?
+                    </StyledIconLine>
```

Comment:
Is this nesting `<p>` tags? I don't think that's valid HTML (and it's quite possible your console is screaming at you - I noticed mine was shouting at me yesterday). 

Putting block-level elements, such as `ul` within `p` tags is also not valid HTML, according to [MDN's article on `p`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p)

Reply:
Fixed

Reply:
Same with this: I'm not seeing the changes. But maybe github is slow? Or maybe you haven't pushed yet? 😅

Reply:
Yeah, there they are!